### ISSUE TITLE
exa displays wrong date format on December 31

### ISSUE THREAD

**Issue by j-tai** [ogham/exa#620]
If the current date is December 31, `<year>`, exa will think that the current year is `<year> + 1` for the purpose of displaying file modification times, so it may choose to use the "day and year" format instead of the "day and time" format, or vice versa.

To reproduce, set your system's current date to December 31, create a file, and run `exa -l`:

```console
$ date
Tue Dec 31 18:17:20 PST 2019
$ echo hello > hello.txt
$ exa -l
.rw-r--r-- 6 jtai 31 Dec  2019 hello.txt
$ ls -l
total 4
-rw-r--r-- 1 jtai jtai 6 Dec 31 18:17 hello.txt
```

Notice that exa displayed the date as "31 Dec 2019", instead of the expected "31 Dec 18:17", even though Dec 31, 2019 is the current day.

**Comment by ariasuni** [comment]
Is this still happening? If not, I believe it’ll be fixed by #867.

**Comment by j-tai** [comment]
Yes, this still happens on master, and it is indeed fixed by #867.

**Comment by ariasuni** [comment]
Closing this, since exa is unmaintained (see https://github.com/ogham/exa/issues/1243), and this has (finally :tada:) fixed in [the active fork eza](https://github.com/eza-community/eza)!
